styles.css

```diff
@@ -397,66 +383,175 @@ h1 {
   max-width: 400px;
 }
 
-@media screen and (max-width: 600px) {
-  .hamberger {
+@media only screen and (min-width: 600px) {
```

Comment:
- **_[OPTIONAL]_** Please also consider using media queries for the Desktop breakpoint where the value for `min-width` property should be in range of `768px` or wider as per the project requirements 👍🏽

![image](https://user-images.githubusercontent.com/43172164/206188581-7de0942b-8ab3-490b-9d89-b2a97c49d9de.png)